backend/src/data/repositories/interview/interview.repository.ts

```diff
@@ -27,13 +28,41 @@ class Interview {
       .execute();
   }
 
+  public getInterviewersByCategoryId(
+    categoryId: number,
+  ): Promise<InterviewsGetInterviewerResponseDto[]> {
+    return this.#InterviewModel
+      .query()
+      .select('interviewer')
+      .where({ categoryId })
+      .where('status', InterviewStatus.COMPLETED)
+      .withGraphJoined(
+        'interviewee(selectIdEmail) as interviewer.userDetails(selectFullName)',
+      )
+      .modifiers({
+        selectIdEmail(builder) {
+          builder.select('id', 'email');
+        },
+        selectFullName(builder) {
+          builder.select('fullName');
+        },
+      })
+      .castTo<InterviewsGetInterviewerResponseDto[]>()
+      .execute();
+  }
+
   public async getById(id: number): Promise<InterviewsByIdResponseDto | null> {
     const interview = await this.#InterviewModel
       .query()
       .select()
-      .withGraphJoined('courseCategory')
-      .withGraphJoined('interviewee')
-      .withGraphJoined('interviewer')
+      .withGraphJoined(
+        '[courseCategory, interviewee(withoutPassword).[userDetails], interviewer(withoutPassword).[userDetails]]',
+      )
+      .modifiers({
+        withoutPassword(builder) {
```

Comment:
let's move modifiers to model, this way we'll be able to reuse them

Reply:
@PavelSukhinin thanks! it was useful for me